docs/features/refout.md

```diff
@@ -83,7 +83,8 @@ That said, even if determinism isn't set, compilation of ref assemblies is [larg
 
 ## MSBuild
 
-* `ProduceReferenceAssembly` (boolean) controls whether to create the item passed to the compiler task (and thus pass `/refout:`). It requires opt-in. It is recommended that `Deterministic` also be set for best result (see details above).
+* `ProduceReferenceAssembly` (boolean) controls whether to create the item passed to the compiler task (and thus pass `/refout:`). It requires opt-in. It is recommended that `Deterministic` also be set for best result (see details above).  Cannot be used in conjunction with `ProduceOnlyReferenceAssembly`
+* `ProduceOnlyReferenceAssembly` (boolean) controls whether to pass `/refonly` to the compiler. It requires opt-in. Cannot be used in conjunction with `ProduceReferenceAssembly`
```

Comment:
Nit: inconsistency on whether or not you put a colon after the option name.

Reply:
`/refout:` takes a path as an option. `/refonly` is just a switch with nothing passed to it.

Reply:
Ah yes